deploy/helm/sumologic/conf/logs/collector/otelwindows/config.yaml

```diff
@@ -0,0 +1,318 @@
+      - from: body.SYSLOG_FACILITY
+        to: attributes.SYSLOG_FACILITY
+        type: copy
+      - from: body._HOSTNAME
+        to: attributes._HOSTNAME
+        type: copy
+      - from: body.PRIORITY
+        to: attributes.PRIORITY
+        type: copy
+      - field: attributes["fluent.tag"]
+        type: add
+        value: EXPR("host." + attributes["_SYSTEMD_UNIT"])
+      - field: body.__CURSOR
+        type: remove
+      - field: body.__MONOTONIC_TIMESTAMP
+        type: remove
+{{- end }}
+
+receivers:
+{{- if .Values.sumologic.logs.container.enabled }}
+  filelog/containers:
+    exclude:
+      - /var/log/pods/{{ template "sumologic.namespace" . }}_{{ template "sumologic.labels.app.sumologic-mock.pod" . }}*/*/*.log
+      {{ if eq .Values.debug.logs.metadata.stopLogsIngestion true }}
+      {{ include "logs.metadata.files.list" . }}
+      {{ end }}
+      {{ if eq .Values.debug.logs.collector.stopLogsIngestion true }}
+      {{ include "logs.collector.files.list" . | nindent 6 }}
+      {{ end }}
+      {{ if eq .Values.debug.logs.otelwindows.stopLogsIngestion true }}
+      {{ include "logs.collector.windows.files.list" . | nindent 6 }}
+      {{ end }}
+      {{ if eq .Values.debug.metrics.metadata.stopLogsIngestion true }}
+      {{ include "metrics.metadata.files.list" . }}
+      {{ end }}
+      {{ if eq .Values.debug.metrics.collector.stopLogsIngestion true }}
+      {{ include "metrics.collector.files.list" . }}
+      {{ end }}
+      {{ if eq .Values.debug.instrumentation.otelcolInstrumentation.stopLogsIngestion true }}
+      {{ include "otelcolInstrumentation.collector.files.list" . }}
+      {{ end }}
+      {{ if eq .Values.debug.instrumentation.tracesGateway.stopLogsIngestion true }}
+      {{ include "tracesGateway.collector.files.list" . }}
+      {{ end }}
+      {{ if eq .Values.debug.instrumentation.tracesSampler.stopLogsIngestion true }}
+      {{ include "tracesSampler.collector.files.list" . }}
+      {{ end }}
+      {{ if eq .Values.debug.events.stopLogsIngestion true }}
+      {{ include "events.collector.files.list" . }}
+      {{ end }}
+
+{{ if lt (int (include "kubernetes.minor" .)) 24 }}
+    ## sets fingerprint_size to 17kb in order to match the longest possible docker line (which by default is 16kb)
+    ## we want to include timestamp, which is at the end of the line
+    ## Not necessary in 1.24 and later, as docker-shim is not present anymore
+    fingerprint_size: 17408
+{{ end }}
+    include:
+      - /var/log/pods/*/*/*.log
+    include_file_name: false
+    include_file_path: true
+    operators:
+      ## Detect the container runtime log format
+      ## Can be: docker-shim, CRI-O and containerd
+      - id: get-format
+        routes:
+          - expr: 'body matches "^\\{"'
+            output: parser-docker
+          - expr: 'body matches "^[^ Z]+ "'
+            output: parser-crio
+          - expr: 'body matches "^[^ Z]+Z"'
+            output: parser-containerd
```

Comment:
Is it even possible to use all of these container runtimes in a Windows Node? Alternatively, is this whole configuration much different than the one for Linux?

Reply:
No, the configuration is the same for containers, except that path regex has differenc format. There will be different configuration for windows events, so analogical to `systemd`/`kubelet` of linux containers

Reply:
Can we move that to a different file and include in both? I don't want to repeat this if it's the same.

Reply:
I moved it to separate template